code/game/machinery/exp_cloner.dm

```diff
@@ -69,6 +71,23 @@
 		H.set_cloned_appearance()
 
 		H.set_suicide(FALSE)
+
+		if(!isnull(H.mind) && prob(defect_chance)) // If we don't have mind, don't set any objectives to it, or it will break the cloner with an infinite loop of exceptions.
+			if(defect_chance < 100)
+				defect_chance = defect_chance + 10
+
+			var/datum/objective/O = new /datum/objective/custom
+			O.team = team
+			O.explanation_text = "Impersonate [clonename]. There can only be one."
+			O.owner = H.mind
+
+			var/datum/antagonist/antag = new /datum/antagonist/custom
```

Comment:
Using a custom antagonist here might get annoying for admins, since they won't show up on admin hud, you should make your own antagonist datum